ChameleonForms.AcceptanceTests/PartialForTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.CodeDom;
+using System.IO;
+using System.Net.Http;
+using System.Text.RegularExpressions;
+using ApprovalTests.Html;
+using ApprovalTests.Reporters;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace ChameleonForms.AcceptanceTests
+{
+    /// <summary>
+    /// Loading partial views is very difficult to test by unit testing.
+    /// </summary>
+    [UseReporter(typeof(DiffReporter))]
+    public class PartialForTests

```

Comment:
Makes sense.

Let's do it in a separate PR though. I'm happy enough that it will postback ok in reviewing the approval test.

--


> On 12 Oct 2015, at 5:28 pm, zabulus <notifications@github.com> wrote:
> 
> In ChameleonForms.AcceptanceTests/PartialForTests.cs:
> 
> > +using System.IO;
> > +using System.Net.Http;
> > +using System.Text.RegularExpressions;
> > +using ApprovalTests.Html;
> > +using ApprovalTests.Reporters;
> > +using NUnit.Framework;
> > +using OpenQA.Selenium;
> > +using OpenQA.Selenium.Support.UI;
> > +
> > +namespace ChameleonForms.AcceptanceTests
> > +{
> > +    /// <summary>
> > +    /// Loading partial views is very difficult to test by unit testing.
> > +    /// </summary>
> > +    [UseReporter(typeof(DiffReporter))]
> > +    public class PartialForTests
> I think also need to add submit test, and check that POST binding works as expected.
> 
> —
> Reply to this email directly or view it on GitHub.
>